test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -0,0 +1,366 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if !ASPNETCORE50
```

Comment:
enable as many of these tests as possible everywhere.  wrap only the `using Moq;` and specific tests using mocks.